### PR TITLE
DOCS: Request new license info in separate doc

### PR DESCRIPTION
**Pull request by jlovejoy**
moving request new license to its own document as discussed in #1211 

put here in DOCS or at top-level??

### CODE REVIEW

**Comment by alandtse** [comment]
If we're going to use a DOCS directory, then it makes sense to move everything but standard docs such as README/CONTRIBUTORS to DOCS. This should be fine in DOCS.

File: DOCS/request-new-license [hunk 1]
```diff
@@ -0,0 +1,28 @@
+
+The SPDX Legal Team appreciates proposals for new free and open source licenses or exceptions to be added to the SPDX License List.  Please make sure to stay engaged with your request and if it is accepted, then help prepare the files necessary to its inclusion on the SPDX License List.
+
+# How to request a new license or exception be added to the SPDX License List
+
+1. Join the legal team mailing list at lists.spdx.org.
+
+1.  Review the [license inclusion principles](DOCS/license-inclusion-principles.md).
```
Comment (mkdolan) [inline DOCS/request-new-license:8]:
```suggestion
2.  Review the [license inclusion principles](DOCS/license-inclusion-principles.md).
```

File: DOCS/request-new-license [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+2.  Check the [SPDX License List](https://spdx.org/licenses/), keeping in mind the [SPDX License List Matching Guidelines](https://spdx.org/spdx-license-list/matching-guidelines) and issues labeled [new license/exception request](https://github.com/spdx/license-list-XML/labels/new%20license%2Fexception%20request) to ensure this license or exception has not been previously requested.
```
Comment (mkdolan) [inline DOCS/request-new-license:10]:
```suggestion
3.  Check the [SPDX License List](https://spdx.org/licenses/), keeping in mind the [SPDX License List Matching Guidelines](https://spdx.org/spdx-license-list/matching-guidelines) and issues labeled [new license/exception request](https://github.com/spdx/license-list-XML/labels/new%20license%2Fexception%20request) to ensure this license or exception has not been previously requested.
```

File: DOCS/request-new-license [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+3. If your license is not already SPDX License List, submit your request via the SPDX Online Tool [Submit New License](https://tools.spdx.org/app/submit_new_license/) using the guidance provided there. Also see the [explanation of the fields contained on the list](DOCS/license-fields.md).
```
Comment (mkdolan) [inline DOCS/request-new-license:12]:
```suggestion
4. If your license is not already SPDX License List, submit your request via the SPDX Online Tool [Submit New License](https://tools.spdx.org/app/submit_new_license/) using the guidance provided there. Also see the [explanation of the fields contained on the list](DOCS/license-fields.md).
```

File: DOCS/request-new-license [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+   * Note: You must have a Github account in order to use this tool/process.
+   * Note: Whether you use the [Submit New License](https://tools.spdx.org/app/submit_new_license/) tool or submit manually via a new issue, you will need to provide *all* of the information as per the form in the tool. Incomplete submissions will delay the process.
+
+4. Submitters are expected to follow and participate in the discussion and answer any request for additional information via the issue and comments. Please note, if submitters are unresponsive for several months, the issue will be closed without making a final decision.
```
Comment (mkdolan) [inline DOCS/request-new-license:16]:
```suggestion
5. Submitters are expected to follow and participate in the discussion and answer any request for additional information via the issue and comments. Please note, if submitters are unresponsive for several months, the issue will be closed without making a final decision.
```

**Review by mkdolan — CHANGES_REQUESTED** [review]
Looks good to me. Suggested a few cleanups.

**Comment by bsdimp** [comment]
lgtm

**Comment by jlovejoy** [comment]
@alandtse 
> If we're going to use a DOCS directory, then it makes sense to move everything but standard docs such as README/CONTRIBUTORS to DOCS. This should be fine in DOCS.

the only thin in the top-level are README, CONTRIBUTING, and RELEASE-NOTES - I think that's what you mean?

File: DOCS/request-new-license [hunk 5]
```diff
@@ -0,0 +1,28 @@
+
+The SPDX Legal Team appreciates proposals for new free and open source licenses or exceptions to be added to the SPDX License List.  Please make sure to stay engaged with your request and if it is accepted, then help prepare the files necessary to its inclusion on the SPDX License List.
+
+# How to request a new license or exception be added to the SPDX License List
+
+1. Join the legal team mailing list at lists.spdx.org.
```
Comment (silverhook) [inline DOCS/request-new-license:6]:
```suggestion
1. Join the [legal team mailing list](https://lists.spdx.org/g/spdx-legal).
```

**Comment by jlovejoy** [comment]
Thanks for all the feedback. I'm going to merge this so then I'll have a link to add for the CONTRIBUTING page. Any further updates can go for a new PR

**Pull request merged**